.github/workflows/continuous-integration.yml

```diff
@@ -45,21 +45,42 @@ jobs:
 
       - name: List changed files
         id: changed-files
+        if: ${{ github.ref != 'refs/heads/master' }}
         run: echo ::set-output name=all_changed_files::$(git diff --name-only origin/master...)
 
+      - name: List changed files - master
+        id: changed-files-master
+        if: ${{ github.ref == 'refs/heads/master' }}
+        run: echo ::set-output name=all_changed_files::$(git diff --name-only HEAD~)
+
       - name: Get app entries for changed files
         id: get-changed-apps
         if: ${{ github.ref != 'refs/heads/master' }}
         run: echo ::set-output name=app_entries::$(node script/github-actions/get-changed-apps.js)
         env:
           CHANGED_FILE_PATHS: ${{ steps.changed-files.outputs.all_changed_files }}
 
+      - name: Get app entries for changed files - master
+        id: get-changed-apps-master
+        if: ${{ github.ref == 'refs/heads/master' }}
+        run: echo ::set-output name=app_entries::$(node script/github-actions/get-changed-apps.js)
+        env:
+          CHANGED_FILE_PATHS: ${{ steps.changed-files-master.outputs.all_changed_files }}
+
       - name: Build
+        if: ${{ github.ref != 'refs/heads/master' }}
         run: yarn build --verbose --buildtype=${{ matrix.buildtype }} ${ENTRY:+"--entry=$ENTRY"}
         timeout-minutes: 30
         env:
           ENTRY: ${{ steps.get-changed-apps.outputs.app_entries }}
 
+      - name: Build
+        if: ${{ github.ref == 'refs/heads/master' }}
+        run: yarn build --verbose --buildtype=${{ matrix.buildtype }} ${ENTRY:+"--entry=$ENTRY"}
+        timeout-minutes: 30
+        env:
+          ENTRY: ${{ steps.get-changed-apps-master.outputs.app_entries }}
```

Comment:
This would turn changed app builds on in `master`, which we don't want to do yet. If we're able to get a single `changed-files` step like I mentioned above, we can remove this and the duplicate `get-changed-apps` step, and get the `app_entries` from that single `get-changed-apps` step